### DISCUSSION TITLE
What is this URL: https://github.sre.pub/

### DISCUSSION THREAD

**Discussion by vjmp** [community/community#16912] [Other Feature Feedback, Questions, & Ideas]
Is this some new service, or why are our repos visible under this domain? Or is this some phishing site?

**Comment by MrKrishnaAgarwal** [comment]
It's a phishing website.
Stay aware and don't go to that site.

**Comment by itzkavindu** [comment]
It is not a website associated with Github. It is a phishing site

**Comment by Nishant1500** [comment]
Yep, it's a phishing site
